schematools/contrib/django/models.py

```diff
@@ -570,3 +570,17 @@ def related_model(self):
         dataset_name, table_name, column_name = [to_snake_case(part)
                                                  for part in self.relation.split(":")]
         return apps.all_models[dataset_name][table_name]
+
+
+ACTIVE_PROFILES = None
+
+
+def get_active_profiles():
+    if ACTIVE_PROFILES is not None:
+        return ACTIVE_PROFILES
+
+    return Profile.objects.all()
+
+
+if hasattr(settings, "CACHE_AUTH_PROFILES_ON_START") and settings.CACHE_AUTH_PROFILES_ON_START:
+    ACTIVE_PROFILES = Profile.objects.all()
```

Comment:
Hier cache je het QuerySet object, maar niet niet de geevalueerde lijst van profielen. Op zich cachen QuerySets hun results, maar all() is daar geloof ik de uitzondering. Ik zou dit veranderen in:
```
ACTIVE_PROFILES = [profile for profile in Profile.objects.all()]
```

Reply:
of:
```python
ACTIVE_PROFILES = list(Profile.objects.all())
```

Reply:
Thnx. Aangepast.